WordPress/Classes/ViewRelated/Post/PostSettingsViewController.m

```diff
@@ -544,28 +581,40 @@ - (void)tableView:(UITableView *)tableView didSelectRowAtIndexPath:(NSIndexPath
         [self showFeaturedImageSelector];
     } else if (cell.tag == PostSettingsRowFeaturedImageAdd) {
         [self showFeaturedImageSelector];
+    } else if (cell.tag == PostSettingsRowShareMessage) {
+        [self showEditShareMessageController];
     } else if (cell.tag == PostSettingsRowGeolocation) {
         [self showPostGeolocationSelector];
     } else if (cell.tag == PostSettingsRowExcerpt) {
         [self showEditExcerptController];
     }
 }
 
+- (NSInteger)numberOfRowsForShareSection
+{
+    if (self.apost.blog.supportsPublicize && self.publicizeConnections.count > 0) {
+        // One row per publicize connection plus an extra row for the publicze message
+        return self.publicizeConnections.count + 1;
+    } else {
```

Comment:
No need for *else* statement, since the return in line 597 will prevent this branch from running